src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -21,9 +21,9 @@ export function getStaticProps(context) {
 
 This guide will help you migrate Amplify JavaScript v5's Storage APIs to the new v6's Storage APIs.
 
-## Deprecated Options
+## Removed Options
```

Comment:
Isn't deprecated the correct verbage? We haven't removed anything yet wrt to `key` usage

Reply:
these are for the V5 ones which was removed in v6 and i added another section for deprecated for the key usage ones.

It was suggestion i got from Harshita to make things clear.

Reply:
Ahh got it, apologies missed that